frontend/src/components/LandingPage/NavBar.js

```diff
@@ -0,0 +1,20 @@
+import { Link } from 'react-router-dom';
+import styles from './LandingPage.module.css';
+import classNames from 'classnames';
+
+function NavBar() {
+  return (
+    <header>
+      <div className={classNames(styles['nav-bar'])}>

```

Comment:
minor adjustment, but we can use the `<nav> </nav>` tag here just to be more semantic instead of a div.